amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -3060,7 +3068,7 @@ Snapshot of the user's full roles/permissions at a specific block
 type ColonyHistoricRole @model {
   """
   Unique identifier for the role snapshot
-  Format: `colonyAddress_domainNativeId_userAddress_blockNumber_roles`
+  Format: `<colonyAddress>_<domainNativeId>_<userAddress>_<blockNumber>_roles`  or `<colonyAddress>_<domainNativeId>_<userAddress>_<blockNumber>_multisig_roles`
```

Comment:
At some point in the past we used to "decompose" id's to extract various data from it _(ie: the user address)_

I don't think we still do it, but it's something to be aware, since that might break with the inclusion of two different "formats" of ids